packages/runtime-core/src/componentEmits.ts

```diff
@@ -160,35 +160,42 @@ export function emit(
     handler = props[(handlerName = toHandlerKey(hyphenate(event)))]
   }
 
+  const results = []
   if (handler) {
-    callWithAsyncErrorHandling(
+    const values = callWithAsyncErrorHandling(
```

Comment:
I wonder if `values` could be an array? If `handler` is not an array, then `values` is not.

Reply:
Based on the definition of `callWithAsyncErrorHandling` it can return an array:
https://github.com/vuejs/core/blob/ae4b0783d78670b6e942ae2a4e3ec6efbbffa158/packages/runtime-core/src/errorHandling.ts#L77-L98

Reply:
It depends on `handler`. If `handler` is a function, then `callWithAsyncErrorHandling` will return the result directly. I don't think `handler` will be an array.